### PR TITLE
PANGOLIN-2442 - Add graphql draft representation to Image

### PR DESCRIPTION
**Pull request by valoriecarli**
Circled back to Image to include ImageInput. 

Screenshot 2023-03-01 at 09 17 25

Thanks @jaikamat for catching me up to speed on this. 🤯

### CODE REVIEW

File: models/image/src/image-draft/transformers.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        ...fields,
+        dimensions: {
+          ...fields.dimensions,
+          __typename: 'DimensionsInput',
+        },
+        __typename: 'ImageInput',
+      };
+    },
+  }),
+};
```
Comment (jaikamat) [inline models/image/src/image-draft/transformers.ts:18]: I _love_ that we only have a graphql transform for this. Really lets consumers know that _this is the one use case_

**Comment by jaikamat** [comment]
Any chance we can include integrating this PR [here](https://github.com/commercetools/test-data/blob/main/models/product-variant/src/product-variant-draft/builder.spec.ts#L83)?

File: models/product-variant/src/product-variant-draft/builder.spec.ts [hunk 2]
```diff
@@ -74,13 +74,14 @@ describe('builder', () => {
...
-             *
-             * TODO: pull in ImageDraft in the generator when it's created
-             */
-            __typename: 'Image',
+            dimensions: expect.objectContaining({
+              w: expect.any(Number),
+              h: expect.any(Number),
+              __typename: 'DimensionsInput',
+            }),
+            __typename: 'ImageInput',
```
Comment (tylermorrisford) [inline models/product-variant/src/product-variant-draft/builder.spec.ts:84]: 💪

**Review by tylermorrisford — APPROVED** [review]
💎

**Review by stephsprinkle — APPROVED** [review]

**Review by jaikamat — APPROVED** [review]

**Pull request merged**